hmmlearn/hmm.py

```diff
@@ -554,12 +555,31 @@ def __init__(self, n_components=1, n_mix=1,
         self.covariance_type = covariance_type
         self.min_covar = min_covar
         self.n_mix = n_mix
+        self.weights_prior_alphas = np.asarray(weights_prior)
+        self.means_prior = {
+            "mus": np.asarray(means_prior[0]),
```

Comment:
In the original version `means_prior[0]` is `means_prior` and `means_prior[1]` is `means_weights`. I think we should avoid breaking API compatibility here.